Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:jessie
+
+RUN mkdir /fitbit-googlefit/
+WORKDIR /fitbit-googlefit/
+
+COPY requirements.txt /fitbit-googlefit/requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . /fitbit-googlefit/
+
+CMD python3 app.py
+
+# Use with the following command: 
+# $ docker run --rm -it -v /path/to/auth/fitbit.json:/fitbit-googlefit/auth/fitbit.json -v /path/to/auth/google.json:/fitbit-googlefit/auth/google.json davinkevin/fitbit-googlefit
```

Comment:
Could you fix `davinkevin/fitbit-googlefit` in this comment?

Reply:
Will you deploy it on the Dockerhub ? If yes, I can put your repo name `praveendath92` as official image.